database/bin/boot

```diff
@@ -71,32 +71,32 @@ INIT_ID=$(etcdctl get $ETCD_PATH/initId 2> /dev/null || echo none)
 echo "database: expecting initialization id: $INIT_ID"
 
 initial_backup=0
-if [[ "$(cat /var/lib/postgresql/9.3/main/initialized 2> /dev/null)" != "$INIT_ID" ]]; then
+if [[ "$(cat /var/lib/postgresql/main/initialized 2> /dev/null)" != "$INIT_ID" ]]; then
```

Comment:
I'm worried about the upgrade path here - for users running the old version of Deis before upgrading database here, their `initialized` file will be in the old location.